optaweb-employee-rostering-standalone/pom.xml

```diff
@@ -51,6 +51,17 @@
   </dependencies>
 
   <build>
+      <pluginManagement>
```

Comment:
This shouldn't be necessary if you bind the `analyze-only` execution to `none` phase.